src/EditorFeatures/Core/Implementation/LineSeparators/AdornmentManager.cs

```diff
@@ -273,6 +279,16 @@ private void UpdateInvalidSpans()
                 var tagSpans = _tagAggregator.GetTags(changedSpan);
                 foreach (var tagMappingSpan in tagSpans)
                 {
+                    if (projectionSnapshot != null)
+                    {
+                        var point = tagMappingSpan.Span.Start.GetPoint(changedSpan.Snapshot, PositionAffinity.Predecessor);
```

Comment:
Nope.  If i do that, then i always get back 'null' no matter if the span is collapsed or not.